pkg/module/command.go

```diff
@@ -23,11 +23,11 @@ func Install(ctx context.Context, dir, repo string, quiet bool, opt types.Regist
 	}
 
 	log.Info("Installing the module from the repository...", log.String("repo", repo))
-	art := oci.NewArtifact(repo, quiet, opt)
+	art := oci.NewArtifact(repo, opt)
 
 	dst := filepath.Join(dir, ref.Context().Name())
 	log.Debug("Installing the module...", log.String("dst", dst))
-	if err = art.Download(ctx, dst, oci.DownloadOption{MediaType: mediaType}); err != nil {
+	if err = art.Download(ctx, dst, oci.DownloadOption{MediaType: mediaType, Quiet: true}); err != nil {
```

Comment:
I'm curious why it's always set to true.

```suggestion
	if err = art.Download(ctx, dst, oci.DownloadOption{MediaType: mediaType, Quiet: quiet}); err != nil {
```

Reply:
That's a typo, [fixed](https://github.com/aquasecurity/trivy/pull/7605/commits/8a960b3a826d3617d88692a8247b81f6bf0ee82b).